src/polytope_blmos.jl

```diff
@@ -272,10 +299,12 @@ end
 
 function check_feasibility(sblmo::ProbabilitySimplexSimpleBLMO, lb, ub, int_vars, n)
     m = n - length(int_vars)
-    if sum(lb) ≤ sblmo.N ≤ sum(ub) + m * sblmo.N
-        return OPTIMAL
+    if !(sum(lb) ≤ sblmo.N ≤ sum(ub) + m * sblmo.N)
```

Comment:
That is not necessary as separate `if` statement, the `else` statement will take care of that.